### DISCUSSION TITLE
role permissions [May 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2112] [Discord Threads]
> 🙋‍♂️ **AK**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### currently no. role permissions will be enhanced in the future  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974670180380704849)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
